gulpfile.js

```diff
@@ -116,14 +116,29 @@ gulp.task( 'translations:download', () => {
 
 // Releasing. -----------------------------------------------------------------
 
-gulp.task( 'changelog:dependencies', () => {
+gulp.task( 'changelog', () => {
 	assertIsInstalled( '@ckeditor/ckeditor5-dev-env' );
 
-	return require( '@ckeditor/ckeditor5-dev-env' )
-		.generateChangelogForSubRepositories( {
-			cwd: process.cwd(),
-			packages: 'packages'
-		} );
+	const devEnv = require( '@ckeditor/ckeditor5-dev-env' );
+	const commonOptions = {
+		cwd: process.cwd(),
+		packages: 'packages'
+	};
+
+	const optionsForDependencies = Object.assign( {}, commonOptions, {
+		skipPackages: [
```

Comment:
I don't like hardcoding just these packages. Can't we use some wildcard? `@ckeditor/ckeditor5-build-*`. Besides, it's inconsistent that this option uses strings while the `scope` option below uses a regexp.